cli.py

```diff
@@ -11,19 +11,29 @@ def check_exit(user_input):
 
 
 def cli_input():
-    user_input = input("> ")
-    if check_exit(user_input):
-        return None
-    return user_input
+    try:
+        user_input = input("> ")
+    except (KeyboardInterrupt, EOFError): # We can add a print command here if you'd like
+        pass
+    else:
+        if check_exit(user_input):
+            return None
+        return user_input
 
 
 def cli_input_multiline():
     user_input = []
     while True:
-        line = input("> ")
-        if line == "SEND":
+        try:
+            line = input("> ")
+        except (KeyboardInterrupt, EOFError): # We can add a print command here if you'd like
+            pass
```

Comment:
Hmm, it does not work se well in multiline mode:
```
$ ./cli.py -m
> Hey
> How[CTRL+C pressed here]Hello! How can I assist you today?
Prompt tokens: 8, Completion tokens: 9, Total price: 0.000 USD
> 
```
I think that solution would be to repeat the same thing from single line mode, i.e. removing `pass` and `break` and adding there empty `print()` and `return None`...

Reply:
Hmm. I'll give it a try and see what I can come up with